README.md

```diff
@@ -0,0 +1,122 @@
+Common clang is a thin wrapper library around clang. Common clang has
+OpenCL-oriented API and is capable to compile OpenCL C kernels to SPIR-V
+modules.
+
+## Build
+
+Source code in this repo can be built in different manners:
+* in-tree as an LLVM project
+* out-of-tree using pre-built LLVM
+
+### In-tree build
+
+Before the build all dependencies must be downloaded and layed out as the
+following:
+
+```
+<workspace>
+`-- llvm
+    |-- tools
+    |   `-- clang
+    `-- projects
+        |-- llvm-spirv
+        `-- common-clang
+```
+
+This can be done using the following commands:
+```
+cd <workspace>
+git clone https://github.com/llvm-mirror/llvm.git -b release_70
```

Comment:
Please drop " -b release_70" in git commands here and below.

Reply:
Done